unity-renderer/Assets/UIComponents/Scripts/Components/Carousel/CarouselComponentView.cs

```diff
@@ -60,7 +60,7 @@ public interface ICarouselComponentView
     /// Adds a new item in the carousel.
     /// </summary>
     /// <param name="item">An UI component.</param>
-    void AddItem(BaseComponentView item);
+    void AddItemWithDotsSelector(BaseComponentView item);
```

Comment:
Please, lets modify the current documentation that we have in notion about the `Carousel` UI Component: https://www.notion.so/decentraland/UI-Components-Reference-f467a8d05d594718a2c12809dcfaf132?p=328f41bff9e2416da2d7372a2d004bfc&pm=s

Reply:
I will. However, I see that it is in the Archive folder:)